sdk/src/main/java/io/radar/sdk/RadarLocationReceiver.kt

```diff
@@ -122,11 +123,7 @@ class RadarLocationReceiver : BroadcastReceiver() {
                     return
                 }
 
-                if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O && !RadarForegroundService.started) {
-                    RadarJobScheduler.scheduleJob(context, location, source)
-                } else {
-                    Radar.handleLocation(context, location, source)
-                }
+                Radar.handleLocation(context, location, source)
```

Comment:
We probably need more time to test this one, since we’rentrying cut this build today